xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/less/messages.less

```diff
@@ -65,7 +65,9 @@ fieldset.xwikimessage { // Used by: Login form, Delete messages
 // Remove space added by last elements inside box, like paragraphs.
 .box > div > *:last-child,
 // Do the same when the box macro is edited in-line.
-.box > div > .xwiki-metadata-container > *:last-child {
+.box > div > .xwiki-metadata-container > *:last-child,
+// Keep the previous selector for backwards compatibility with hard coded boxes that didn't get updated
+.box > *:last-child, .box > .xwiki-metadata-container > *:last-child {
```

Comment:
I doubt that the second rule here is required as those hard coded boxes shouldn't be editable. But then again, I guess it doesn't hurt - and after the previous experience, I guess we better keep it.